atomic_reactor/constants.py

```diff
@@ -101,3 +101,11 @@
 HTTP_CLIENT_STATUS_RETRY = (408, 500, 502, 503, 504)
 # requests timeout in seconds
 HTTP_REQUEST_TIMEOUT = 600
+
+
+# Media types
+MEDIA_TYPE_DOCKER_V1 = "application/json"
```

Comment:
What about turning all this into a single data structure? It'll make usage a bit simpler (one thing to import)
```python
MEDIA_TYPES = {
   'v1': 'application/json',
   'v2schema2': 'application/vnd.docker.distribution.manifest.v1+json'
   ....
```

or if we're concerned with mutability:
```python
_MediaTypes = namedtuple('MediaTypes', 'v1,v2schema2,...')
MEDIA_TYPES = _MediaTypes(v1='application/json', ...)
```

Reply:
I like the idea, but in most places only one or two constants are used, so it would just make things look longer with no real improvement.

@twaugh @rcerven thoughts?

Reply:
I'm fine with either way.

Reply:
Let's drop this suggestion then.